packages/pipelines/src/commands/reviewapps/enable.ts

```diff
@@ -2,47 +2,66 @@ import {Command, flags} from '@heroku-cli/command'
 import * as Heroku from '@heroku-cli/schema'
 import cli from 'cli-ux'
 
-export default class ReviewappsEnable extends Command {
+export default class ReviewappsEnable2 extends Command {
   static description = 'enable review apps and/or settings on an existing pipeline'
 
   static examples = [
-    '$ heroku reviewapps:enable -p mypipeline -a myapp --autodeploy --autodestroy',
+    `$ heroku reviewapps:enable -p mypipeline --autodeploy --autodestroy
+`,
```

Comment:
you could remove the extra new line and change `` to '' so tslint doesn't yell at you.